xml/ha_storage_protection.xml

```diff
@@ -216,6 +216,18 @@
     <para>An SBD device can be shared between different clusters, as
      long as no more than 255 nodes share the device. </para>
    </listitem>
+   <listitem>
+     <para>
+       When using more than one SBD device, all nodes must have a slot in all SBD devices.
+       Fencing does not work with an asymmetric SBD setup.
```

Comment:
Out of my head, I just came up with a small bash scrpt for users, not sure if appropriate as for DOC per say. 

``` bash
SBD_DEVICE="/dev/disk/by-partlabel/sbd-155;/dev/disk/by-partlabel/sbd-155-1"
crm cluster run " for i in `echo $SBD_DEVICE|tr ';' ' ' `; do sbd -d $i list; done "
```

Similarly, the sbd "dump" command to verify timeouts

``` bash
SBD_DEVICE="/dev/disk/by-partlabel/sbd-155;/dev/disk/by-partlabel/sbd-155-1"
crm cluster run " for i in `echo $SBD_DEVICE|tr ';' ' ' `; do sbd -d $i dump; done "
```